previewer.js

```diff
@@ -25,8 +25,13 @@ function Previewer() {
 
 	// Define function to close preview
 	this.closeWindow = function () {
-		document.getElementById("previewer").style.display = "none";
-		document.getElementById("previewer-img").src = "";
+		setTimeout(function(){
+				document.getElementById("previewer").style.display = "none";
+				document.getElementById("previewer-img").style.animation = 'pop-animation 0.5s';
+			},400);
```

Comment:
What's the purpose of this timeout function and how does it work? I'm asking cause you seem to have added it to the close button function.